README.md

```diff
@@ -1,7 +1,109 @@
 # Pipeline Stage View Plugin
 
-This plugin provides a graphical swimlane view of pipeline stage execution (as well as a build history of the stages). 
+# Stage View: 
+
+## ![](docs/images/who-broke-it.png)
+
+## ![](docs/images/green-and-mean.png)
+
+When you have complex builds Pipelines, it is useful to be able to see
```

Comment:
Suggest: 
When you have complex build Pipelines, it is useful to be able to see

Replaces `builds` with `build`